pkgs/tools/archivers/p7zip/default.nix

```diff
@@ -49,6 +57,11 @@ stdenv.mkDerivation rec {
     description = "A port of the 7-zip archiver";
     platforms = stdenv.lib.platforms.unix;
     maintainers = [ stdenv.lib.maintainers.raskin ];
+    knownVulnerabilities = [
+      # p7zip is abandoned, according to this thread on its forums:
+      # https://sourceforge.net/p/p7zip/discussion/383043/thread/fa143cf2/#1817
+      "p7zip is abandoned and may not receive important security fixes"
```

Comment:
Are there any `known` security vulnerabilities though right now?

Reply:
Not as far as I'm aware.

Reply:
https://discourse.nixos.org/t/unmaintained-software-when-do-we-deprecate-software/7030/6